app/scheduler/pyproject.toml

```diff
@@ -32,14 +32,14 @@ uWSGI = "^2.0.20"
 cloudstorage = {extras = ["google"], version = "^0.11.0"}
 kubernetes = "^18.20.0"
 boto3 = "^1.20.26"
-setuptools = "^60.0.0"
+setuptools = "*"
```

Comment:
@simeoncarstens, I hope that this doesn't cause any trouble with respect to https://github.com/tweag/chainsail/pull/437. Having "^60.0.0" meant that poetry could not solve the constraints.

Reply:
If it works, I guess it's fine. The lock file shows that the `setuptools` dependency resolves to v67.4, so very recent. And as such it's pinned in the lock file, but having no version bounds at all in `pyproject.toml` looks weird somehow.